### DISCUSSION TITLE
Any Plans to Support OGC I3S - Indexed 3D Scene Layers?

### DISCUSSION THREAD

**Discussion by tech-maven-geospatial** [TerriaJS/terriajs#4894] [Q&A] [answered]
We've got many of users (and data providers) that have data in i3s NOT 3DTILES  (yes they are both OGC Standards)
**are there plans to support i3s in the future?**
or do you know of a converter?
I heard of an OGC 3D portrayal service that can output to 3DTILES from i3s but not seen any implementations of this Server/Service.
https://www.ogc.org/standards/3dp

**Comment by KeyboardSounds** [comment, marked as answer]
We actually have an i3s to 3dtiles converter that we've developed and use extensively internally. It's primarily a command line utility, but we also have a server version that can convert datasets "on the fly" as they are requested by a user in the browser.

However, we don't have concrete plans to release it, although we want to eventually. It's looking unlikely at this point that we'll make it available under an open source licence— we love open source, and the TerriaJS open source community is incredibly important to us, but we have to financially support ourselves somehow! If we make any decisions, I'll be sure to keep you updated.

To help inform our discussion, it would be great to hear more about the sort of use cases you'd have for i3s in Terria. Would you be interested in licensing an i3s to 3dtiles converter if it was available?

**Comment by techmavengeospatial** [comment]
Yes would pay for it
If it made financial sense

We would be interested in the server side part with API or purchase of
command line tool
Thanks
Jordan


Jordan Bess
Tech Maven Geospatial
Sr. Geospatial Data Engineer
Chief Software Solutions Architect
